README.md

```diff
@@ -304,6 +304,19 @@ address("/price/usd/gbp")
   .post({foo:"bar"})
 ```
 
+### Adding timeout
+When requesting a resource address will by default wait 30 seconds for a response from the resource, before responding with a ```408 Request Timeout``` error. By calling the ```timeout``` method on the request, it is possible to increase or decrease this timeout period. The following example shows how to increase the timeout from the default 30 seconds to 60:
+
+```javascript
+// increase the timeout to 60000ms (60 sec) for request to resolve
+address("/price/usd/gbp")
+  .timeout(60000)
```

Comment:
I still think we should change this API to accept seconds, rather than milliseconds. As a user, you could still increase granularity by just using fractions (e.g. `0.5` for a half second) if we just multiply the parameter value by `1000`. There's a technical risk of inaccuracies I suppose because we'd be using floats, but it's not like the timeout is always accurate anyway so I don't think it matters.

Reply:
Updated `timeout` method to accept seconds instead of milliseconds.